tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeSpacetimeQuantities.cpp

```diff
@@ -165,14 +168,18 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.SpacetimeDecomp",
   MAKE_GENERATOR(generator);
   std::uniform_real_distribution<> distribution(-0.1, 0.1);
 
-  auto expected_spacetime_metric =
-      make_with_random_values<tnsr::aa<DataVector, 3, Frame::Inertial>>(
-          make_not_null(&generator), make_not_null(&distribution),
-          used_for_size);
-  get<0, 0>(expected_spacetime_metric) += -1.;
-  for (size_t i = 1; i <= 3; ++i) {
-    expected_spacetime_metric.get(i, i) += 1.;
-  }
+  const auto expected_spacetime_metric = [&]() {
```

Comment:
you should only capture what you need, `&generator, &distribution, &used_for_size`

Reply:
done